docker-compose.yml

```diff
@@ -20,7 +20,7 @@ services:
     env_file:
       - .env
     ports:
-      - "5432:5432"
+      - "5438:5432"
```

Comment:
why?

Reply:
because in local I have database with "5432:5432" so I can't connect to docker in IDE with 5432